backend/tracim_backend/command/database.py

```diff
@@ -139,7 +139,7 @@ def take_action(self, parsed_args: argparse.Namespace) -> None:
             except FileNotFoundError:
                 print(
                     'Warning ! Can delete depots file, is depot path correctly'
-                    'configured ?'
+                    ' configured ?'
```

Comment:
removing this space is an error: this will result in returning `Warning ! Can delete depots file, is depot path correctlyconfigured ?`

By the way, there are no speces before punctuation in english.

Reply:
"removing this space is an error"
Right, that why this PR space here.

Reply:
sorry, I have read the PR in the inverse sens :-(

Reply:
I'm a bit tired :-s